### PR TITLE
Add E2E scenarios for common libs

### PR DESCRIPTION
**Pull request by pdthummar**
## Description
Adding E2E scenarios for common libs

Fixes #

---

### PR information

- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] If applicable, the PR references the bug/issue that it fixes in the description.
- [x] New Unit tests were added for the changes made and CI is passing.

### Quality of Code and Contribution Guidelines
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-functions-python-worker/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: tests/endtoend/test_http_functions.py [hunk 1]
```diff
@@ -112,14 +113,106 @@ class TestHttpFunctionsStein(TestHttpFunctions):
...
+
+    This test class will spawn a webhost from your <project_root>/build/webhost
+    folder and replace the built-in Python with azure_functions_worker from
+    your code base. Since the Http Trigger is a native suport from host, we
+    don't need to setup any external resources.
+
+    this file is more focus on testing the E2E flow scenarios.
+    """
+
+    def setUp(self):
```
Comment (gavin-aguiar) [inline tests/endtoend/test_http_functions.py:140]: You don't need the setup and teardown method

File: tests/endtoend/test_http_functions.py [hunk 2]
```diff
@@ -112,14 +113,106 @@ class TestHttpFunctionsStein(TestHttpFunctions):
...
+    def get_script_dir(cls):
+        return testutils.E2E_TESTS_FOLDER / 'http_functions' / \
+                                            'common_libs_functions'
+
+    @testutils.retryable_test(3, 5)
+    def test_numpy_should_return_ok(self):
+        r = self.webhost.request('GET', 'numpy_func',
+                                 timeout=REQUEST_TIMEOUT_SEC)
+
+        self.assertTrue(r.ok)
```
Comment (gavin-aguiar) [inline tests/endtoend/test_http_functions.py:159]: Assert the value of response of that function. 
Same with all other tests below

File: tests/endtoend/http_functions/common_libs_functions/common_libs_functions_stein/function_app.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    res = "array: {}".format(np.array([1, 2, 3], dtype=complex))
+
+    return func.HttpResponse(res)
+
+
+@app.route(route="opencv_func")
+def opencv_func(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info('Python HTTP trigger function processed a request.')
+
+    img = cv2.imread("test.png", cv2.IMREAD_COLOR)
```
Comment (gavin-aguiar) [inline tests/endtoend/http_functions/common_libs_functions/common_libs_functions_stein/function_app.py:41]: You don't need to tests the library. To keep it simple just return and assert any attribute of opencv

**Review by gavin-aguiar — CHANGES_REQUESTED** [review]

File: tests/endtoend/test_http_functions.py (hunk 1 above)
Comment (pdthummar) [reply tests/endtoend/test_http_functions.py]: updated. Thanks

File: tests/endtoend/test_http_functions.py (hunk 2 above)
Comment (pdthummar) [reply tests/endtoend/test_http_functions.py]: updated as per recommendation. Thanks

File: tests/endtoend/http_functions/common_libs_functions/common_libs_functions_stein/function_app.py (hunk 3 above)
Comment (pdthummar) [reply tests/endtoend/http_functions/common_libs_functions/common_libs_functions_stein/function_app.py]: updated. Thanks

**Review by vrdmr — APPROVED** [review]

**Review by gavin-aguiar — APPROVED** [review]

**Pull request merged**